data/mods/My_Sweet_Cataclysm/sweet_monster_factions.json

```diff
@@ -57,5 +57,10 @@
     "name": "licorice_snakes",
     "neutral": [ "bot", "plant", "nether", "human", "crackers", "cookies" ],
     "by_mood": [ "insect", "player", "animal", "gummy_bears", "chewgum" ]
+  },
+  {
+    "type": "MONSTER_FACTION",
+    "name": "candy_pumpkin",
+    "by_mood": [ "crackers", "cookies", "gummy_bears", "chewgum" ],
```

Comment:
it should not be hostile to crackers and cookies since they're made of a different thing are thus not so edible for the candy pumpkin. The rule is that candies attack other similar candies

Reply:
You also probably need to add them to the "friendly" list of zombie so that zombies ignore them